cob_hardware_config/raw3-2/config/local_costmap_params.yaml

```diff
@@ -12,18 +12,34 @@ width: 5.0
 height: 5.0
 resolution: 0.07
 
-# footprint and range
-obstacle_range: 3.0 #[m]
-max_obstacle_height: 2.0 #[m]
-raytrace_range: 3.0 #3.0 #[m]
+# footprint
 footprint: [[0.52, 0.3], [0.52, -0.3], [-0.52, -0.3],[-0.52, 0.3]] #[m]
 footprint_padding: 0.02 #[m]
-inflation_radius: 0.55 #[m]
 
-# sensor setup
-observation_sources: laser_scan_sensor_front laser_scan_sensor_rear
-laser_scan_sensor_front: {sensor_frame: base_laser_front_link, data_type: LaserScan, topic: scan_front, marking: true, clearing: true}
-
-laser_scan_sensor_rear: {sensor_frame: base_laser_rear_link, data_type: LaserScan, topic: scan_rear, marking: true, clearing: true}
+# costmap layers
+plugins:
+  - {name: obstacle_layer, type: "costmap_2d::ObstacleLayer"}
+  - {name: inflation_layer, type: "costmap_2d::InflationLayer"}
```

Comment:
do these changes fix https://github.com/ipa320/cob_robots/issues/231?
Please apply to all robots....